### PR TITLE
bug 1164503 - Upgrade Prism.js, add Rust language

### PR DESCRIPTION
**Pull request by darkwing**
Upgrades Prism to modern version and adds Rust as a language.

Marking as "not ready" at the request of @openjck .  Tested using https://developer.allizom.org/en-US/docs/User:stephaniehobson:code

### CODE REVIEW

**Comment by openjck** [comment]
I hate to be a pain, but can I be selfish and ask that we hold on this until django-pipeline lands. Rebases are going to be a nightmare. :frown:

**Comment by darkwing** [comment]
Sure thing @openjck

**Comment by darkwing** [comment]
Note to self:  strip any  `<pre><code><code>` tags of `language-` classnames to avoid the error triggered on @stephaniehobson's sample page

File: kuma/static/js/libs/prism/prism.css [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+code[class*="language-"],
+pre[class*="language-"] {
+	color: black;
+	text-shadow: 0 1px white;
+	font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+	direction: ltr;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+	word-wrap: normal;
+	line-height: 1.5;
+
+	-moz-tab-size: 4;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+	-o-tab-size: 4;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+	tab-size: 4;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+
+	-webkit-hyphens: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+	-moz-hyphens: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+	-ms-hyphens: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+	hyphens: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+	padding: inherit 0;
+	margin-top: 1em; /* Same as .prism’s padding-top */
+
+	background: hsla(24, 20%, 50%,.08);
+	background: -moz-linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	background: -webkit-linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	background: -o-linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	background: linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	
+	pointer-events: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

**Comment by darkwing** [comment]
OK, made that update.  Once @openjck is ready, this is ready for review.

File: kuma/static/js/libs/prism/prism.css [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+	counter-reset: linenumber;
+}
+
+pre.line-numbers > code {
+	position: relative;
+}
+
+.line-numbers .line-numbers-rows {
+	position: absolute;
+	pointer-events: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+
+}
+
+	.line-numbers-rows > span {
+		pointer-events: none;
```
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

**Comment by darkwing** [comment]
Legit test failure, updating...

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

**Comment by openjck** [comment]
Needs a rebase. Thanks for letting me be selfish and get django-pipeline through first. :smile:

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

**Comment by darkwing** [comment]
Rebased!

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

**Comment by openjck** [comment]
> +259 −3,939

:heart:

**Comment by openjck** [comment]
The code changes look good to me, and syntax highlighting continues to look good for multiple languages in both development and production mode.

:cake:

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)

File: kuma/static/js/libs/prism/prism.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 4 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS3 tab-size not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 5 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 6 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 7 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 8 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS Hyphenation not supported by: Chrome (43,44), Opera (29,30), IE Mobile (10,11)

File: kuma/static/js/libs/prism/prism.css (hunk 9 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 10 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/js/libs/prism/prism.css (hunk 11 above)
Comment (mdn-services) [inline kuma/static/js/libs/prism/prism.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

**Pull request merged**